### DISCUSSION TITLE
[docs] Rename "Minimal setup" guide

### DISCUSSION THREAD

**Discussion by ArmandPhilippot** [chakra-ui/panda#2019] [Ideas]
### Subject

Guides

### Description

I wanted to experiment with PandaCSS, coming from Vanilla Extract it's not too complicated. However, the default presets bother me (I prefer to use a custom theme).

From my understanding, in addition to an empty array for the `preset` key, I need to set `eject` to `true`. It seems to work. However, I can't use my custom tokens when writing styles. I explored the documentation (while omitting "Minimal setup") and I haven't found any guidance...

From the issues, I finally found the solution. Thanks to #1452 I understand my issue. The word "minimal" reminded me of "Minimal setup" and indeed the explanation is here.

For me "Minimal setup" is not clear. When I see that, I'm thinking about a minimal setup to have PandaCSS working. I don't see any connection with `eject`. I'm not sure how the section should be renamed but I think "Minimal setup" is confusing. Some ideas:
* Custom presets (since I crossed `definePreset` it makes more sense to me)
* Enable eject

Another solution would be to add a mention in the [Theme](https://panda-css.com/docs/customization/theme) page. If someone is interested in making its own theme, the `eject` option is more likely to interest him and the explanation about the utilities should be in the same place (or linked to another page).

Note: I am not a native English speaker; maybe that affects my understanding.

Maybe it's another problem, but the auto-completion is also a bit frustrating with this solution. I'm seeing some suggestions that don't appear in my tokens:
* I removed the presets (`presets: []`),
* I added `eject: true`,
* I have created some tokens for my font sizes (`sm`, `md`, `lg`, ...) in a separated file,
* I added something like:

```ts
import { theme } from './theme';

export const pandaPreset = definePreset({
  theme,
  utilities: {
    fontSize: {
      values: 'fontSizes',
    },
  },
});
```
And I'm using this preset in `panda.config.ts`:

```ts
import { defineConfig } from '@pandacss/dev';
import { pandaPreset } from './src/styles/panda-preset';

export default defineConfig({
  eject: true,
  exclude: [],
  include: ['./src/**/*.{astro,js,jsx,ts,tsx}'],
  logLevel: 'info',
  minify: true,
  outdir: 'styled-system',
  preflight: false,
  presets: [pandaPreset],
  shorthands: true,
  strictTokens: true,
  strictPropertyValues: true,
});
```

Now, in the auto-completion (VS Code intellisense), I see both my tokens and some others keywords like `large`, `larger`, `x-large`...

![example](https://github.com/chakra-ui/panda/assets/59021693/164fc293-886f-4446-8a9e-8646d0163fb9)

Is there one more step to disable these suggestions? (note: I haven't tried deactivating all my VS Code extensions; I don't see why it would come from there since it's the first time I've seen something like that.)

Thanks!

Edit: I have only test the auto-completion in `cva` function.

**Comment by anubra266** [comment]
@ArmandPhilippot For the autocomplete, those other ones aren't tokens, they're the built in values in native css
https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#syntax

**Comment by ArmandPhilippot** [comment]
Ah I didn't understand that the built-in values were taken over. I thought they were aliases (since it is not a CSS file). Thanks for the explanation! That part is not a big deal anyway.
